book/src/bridge-module/opaque-types/README.md

```diff
@@ -213,31 +213,55 @@ mod ffi {
     extern "Rust" {
         #[swift_bridge(Equatable)]
         type RustPartialEqType;
-
-        #[swift_bridge(init)]
-        fn new(num: u32) -> RustEquatableType;
     }
 }
 
 #[derive(PartialEq)]
-struct RustPartialEqType(u32);
```

Comment:
Oh sorry we should keep this. I just meant to get rid of the function new impl and bridge.